corehq/apps/app_manager/xpath.py

```diff
@@ -29,7 +30,7 @@ def dot_interpolate(string, replacement):
     return re.sub(DOT_INTERPOLATE_PATTERN, repl, string)
 
 
-def interpolate_xpath(string, case_xpath=None):
+def interpolate_xpath(string, case_xpath=None, module=None):
```

Comment:
I felt this was particularly ugly, but needed the module in this context.  Open to other suggestions for handling this without having to add more arguments to the function.